src/main/java/org/broadinstitute/hellbender/utils/test/SerializationTestUtils.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.hellbender.utils.test;
+
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.serializer.KryoSerializer;
+import org.apache.spark.serializer.SerializerInstance;
+import org.broadinstitute.hellbender.utils.Utils;
+import scala.reflect.ClassTag;
+import scala.reflect.ClassTag$;
+
+public final class SerializationTestUtils {
```

Comment:
This class is copied from gatk-protected, where it lives in a different package. Can you create a ticket in gatk-protected to delete their copy the next time they rev their gatk dependency, and self-assign?